### PR TITLE
Add Collections for RISCV-Cores

### PR DESCRIPTION
**Pull request by carlosdelfino**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by carlosdelfino** [comment]
I didn't understand the problem with the image, it says it must be empty, I've tried it in three different ways and the error is the same.
I ended up removing it in this last attempt.

**Comment by Quang-102** [comment]
Như nào nhỉ 🙄??

**Review by BlakeWilliams — APPROVED** [review]

File: collections/riscv-cpus/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ - ultraembedded/riscv
+display_name: Risc-V CPUs
+created_by: carlosdelfino 
+---
+
+The market for processors and microcontrollers is increasingly heated, and with the launch of the ISA (Instruction Set Achitecture) RISC-V, an open specification, it opens up a new opportunity for those who want to act either by researching or collaborating with new processors and microcontrollers.
+
+Those who master the synthesis of Hardware with FPGA, can also propose more concretely new approaches for microcontrollers taking advantage of codes already written for the new architecture.
+
+In this collection I try to present some renowned Colors, and open the opportunity for other colleagues to collaborate with their suggestions.
```
Comment (BlakeWilliams) [inline collections/riscv-cpus/index.md:54]: Overall this LGTM, but is `Colors` the correct word here? (I'm not super familiar with riscv)

Comment (carlosdelfino) [inline collections/riscv-cpus/index.md]: correct is cores. 
typing corrections send to new push request.

**Review by carlosdelfino — COMMENTED** [review]
typing correction.

**Review by AndreaGriffiths11 — APPROVED** [review]
LGTM, thanks for contributing ✨

**Pull request merged**